3d_armor/README.md

```diff
@@ -350,7 +358,42 @@ Durability is determined by the value assigned to the group ***armor_use***. The
  All diamond armor items have an  ***armor_use=200***;
  
 	65535/2000 = 327.6 (327)   
- After 327 uses(hits) the armor item will break.   
+ After 327 uses(hits) the armor item will break. 
+
+### Armor Material 
+The material the armor is made from is defined by adding the material to the end of registered armor item name. It is very important the material is the last item in the registered item name and it is preceeded by an "_" eg "_materialname".
+The material name is what 3d_armor uses to determine if a player is wearing a set of armor. To recieve the set bonus all items worn must be made of the same material.
+
+So to get a set bonus under the default set settings the players armor items listed below must be made of the same material:
+head - Helmet   
```

Comment:
`* head - Helmet`

And similar for the rest of this list. Please trim tailing spaces.